python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_logistic.py

```diff
@@ -84,28 +124,96 @@ def logisticModel(X):
+#
+# +-------+----------------+
+# | Index | Input variable |
+# +=======+================+
+# | 0     | t1             |
+# +-------+----------------+
+# | 1     | t2             |
+# +-------+----------------+
+# | ...   | ...            |
+# +-------+----------------+
+# | 21    | t22            |
+# +-------+----------------+
+# | 22    | a              |
+# +-------+----------------+
+# | 23    | c              |
+# +-------+----------------+
+#
+# +-------+-----------+
+# | Index | Parameter |
+# +=======+===========+
+# | ∅     | ∅         |
+# +-------+-----------+
+#
+# **Table 1.** Indices and names of the inputs and parameters of the physical model.
+#
+print("Physical Model Inputs:", lm.model.getInputDescription())
+print("Physical Model Parameters:", lm.model.getParameterDescription())
+
+# %%
+# In order to perform calibration, we have to define a parametric model,
+# with observed inputs and parameters to calibrate.
+# In order to do this, we create a :class:`~openturns.ParametricFunction` where the parameters
+# are `a` and `c` which have the indices 22 and 23 in the physical model.
+#
+# +-------+----------------+
+# | Index | Input variable |
+# +=======+================+
+# | 0     | t1             |
+# +-------+----------------+
+# | 1     | t2             |
+# +-------+----------------+
+# | ...   | ...            |
+# +-------+----------------+
+# | 21    | t22            |
+# +-------+----------------+
+#
+# +-------+-----------+
+# | Index | Parameter |
+# +=======+===========+
+# | 0     | a         |
+# +-------+-----------+
+# | 1     | c         |
+# +-------+-----------+
+#
+# **Table 2.** Indices and names of the inputs and parameters of the parametric model.
+#
+
 # %%
 logisticParametric = ot.ParametricFunction(logisticModelPy, [22, 23], thetaPrior)
+print("Parametric Model Inputs:", logisticParametric.getInputDescription())
+print("Parametric Model Parameters:", logisticParametric.getParameterDescription())
 
 # %%
 # Check that we can evaluate the parametric function.
 
 # %%
 populationPredicted = logisticParametric(timeObservations.asPoint())
-populationPredicted
+print(populationPredicted[:5])
 
 # %%
-graph = ot.Graph("", "Time (years)", "Population (Millions)", True, "topleft")
+graph = ot.Graph(
+    "Before calibration", "Time (years)", "Population (Millions)", True, "topleft"
+)
 # Observations
 cloud = ot.Cloud(timeObservations, populationObservations)
 cloud.setLegend("Observations")
-cloud.setColor("blue")
+cloud.setPointStyle(
+    ot.ResourceMap.GetAsString("CalibrationResult-ObservationPointStyle")
+)
 graph.add(cloud)
 # Predictions
 cloud = ot.Cloud(timeObservations.asPoint(), populationPredicted)
 cloud.setLegend("Predictions")
-cloud.setColor("green")
+cloud.setPointStyle(ot.ResourceMap.GetAsString("CalibrationResult-PriorPointStyle"))
 graph.add(cloud)
+graph.setColors(ot.Drawable.BuildDefaultPalette(2))
 view = viewer.View(graph)
 
 # %%
```

Comment:
The next line is out of reach of Github suggestions, but contains a typo: "seem to fade" should be replaced with "seems to fade".

Reply:
Ok. Fixed.